doc/administrator/mapfile.rst

```diff
@@ -61,16 +55,18 @@ resolution into account.
 WFS GetFeature
 --------------
 
-Layers involved in the ``box query`` and ``query builder`` features should
-support WFS GetFeature. To support WFS GetFeature a ``LAYER`` should define
-a template::
+Layers involved in the ``query`` (box and point), the ``filters``, and the WFS permalink features should
```

Comment:
Are we saying:
"Layers that you want to use in a ``query`` (box and point), in ``filters``, or for WFS permalink features must support WFS GetFeature."
?

Reply:
Yes, will be updated :-)